content/applications/finance/accounting/reporting/year_end.rst

```diff
@@ -2,115 +2,112 @@
 Year-end closing
 ================
 
-Before going ahead with closing a fiscal year, there are a few steps one
-should typically take to ensure that your accounting is correct, up to
-date, and accurate:
-
-- Make sure you have fully reconciled your **bank account(s)** up to
-  year end and confirm that your ending book balances agree with
-  your bank statement balances.
-
--  Verify that all **customer invoices** have been entered and approved.
-
--  Confirm that you have entered and agreed all **vendor bills**.
-
--  Validate all **expenses**, ensuring their accuracy.
-
-- Corroborate that all **received payments** have been entered and
-  recorded accurately.
+Year-end closing is vital for maintaining financial accuracy, complying with regulations, making
+informed decisions, and ensuring transparency in reporting.
 
 .. _year-end/fiscal-years:
 
-Manage fiscal years
-===================
+Configuration
+=============
 
-In most cases, the fiscal years last 12 months. If it is your case, you
-just have to define what is the last day of your fiscal year in the
-accounting settings. By default, it is set on the 31st December.
+Fiscal years
+------------
 
-However, there might be some exceptions. For example, if it is the first
-fiscal year of your business, it could last more or less than 12 months.
-In this case, some additional configuration is required.
+A standard fiscal year lasts 12 months and often ends on December 31st. However, both its duration
+and end date can vary due to cultural, administrative, and economic considerations.
```

Comment:
- not sure if it's okay to say "standard" since everything is valid. It's just less common. --> start with "Bu default, the fiscal year is set to last twelve months and end on December 31st.
- write "twelve"